### PR TITLE
added un_ga dataset

### PR DESCRIPTION
**Pull request by param087**
Hi :hugs:, This is a PR for [United nations general assembly resolutions: A six-language parallel corpus](http://opus.nlpl.eu/UN.php) dataset

### CODE REVIEW

File: datasets/un_ga/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+- found
+language_creators:
+- found
+languages:
+- ar
+- en
+- es
+- fr
+- ru
+- zh
```
Comment (lhoestq) [inline datasets/un_ga/README.md:12]: The languages depend on the config name. Could you format the languages accordingly ?
For example
```
languages:
  ar-to-es:
  - ar
  -es
```

**Review by lhoestq — COMMENTED** [review]
Awesome thanks !

I just left one comment about the language tags

File: datasets/un_ga/README.md (hunk 1 above)
Comment (param087) [reply datasets/un_ga/README.md]: @lhoestq Added language tags and BibTex :smile:

**Comment by lhoestq** [comment]
Looks like this PR includes changes about many other files than the ones for un_ga

Can you create another branch an another PR please ?

File: datasets/un_ga/README.md [hunk 2]
```diff
@@ -4,12 +4,51 @@ annotations_creators:
...
 languages:
-- ar
-- en
-- es
-- fr
-- ru
-- zh
+- ar-to-en
+  - ar
+  - en
```
Comment (lhoestq) [inline datasets/un_ga/README.md:9]: Can you remove the `-` in front of config names ? And add `:` after each config name ?
For example
```
  ar-to-en:
  - ar
  - en
```

**Review by lhoestq — COMMENTED** [review]
Nice :)

File: datasets/un_ga/un_ga.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                x = x.strip()
+                y = y.strip()
+                result = (
+                    sentence_counter,
+                    {
+                        "id": str(sentence_counter),
+                        "translation": {lang1: x, lang2: y},
+                    },
+                )
+                sentence_counter += 1
```
Comment (lhoestq) [inline datasets/un_ga/un_ga.py:156]: You can remove this line

sentence_counter is already incremented thanks to enumerate

**Review by lhoestq — COMMENTED** [review]
Looks all good :) 

Feel free to ping me when you have a new clean PR

**Comment by param087** [comment]
@lhoestq, Thank you for suggestions. I have made the changes and raised the new PR https://github.com/huggingface/datasets/pull/1569.

**Pull request closed without merging**